### PR TITLE
Revert "libtorrent-rasterbar: update to 1.2.9."

### LINKED ISSUE #24512: qbittorrent-nox: symbol lookup error on launch
### System

* xuname:  
  Void 5.7.17_1 x86_64 GenuineIntel notuptodate rrFF
* package:  
  qbittorrent-nox-4.2.5_2

### Expected behavior

The program launches

### Actual behavior

```
qbittorrent-nox: symbol lookup error: qbittorrent-nox: undefined symbol: _ZNK10libtorrent5entry4dictB5cxx11Ev
```

### Steps to reproduce the behavior

```
$ sudo xbps-install -S qbittorrent-nox
...

$ qbittorrent-nox
qbittorrent-nox: symbol lookup error: qbittorrent-nox: undefined symbol: _ZNK10libtorrent5entry4dictB5cxx11Ev
```

Additional info:

I have 2 libtorrent libraries in /usr/lib:
```
/usr/lib/libtorrent-rasterbar.so.10.0.0
/usr/lib/libtorrent-rasterbar.so.10
```

`qbittorrent-nox` is linking against `libtorrent-rasterbar.so.10`

### PR DESCRIPTION
**Pull request by ericonr**
This reverts commit 216fdd4f1f998447fd8d2cb4ed22cc216b193fac.

Fixes #24512

@pullmoll

### CODE REVIEW

**Comment by ericonr** [comment]
This is a revert because rebuilding qbittorrent with the new version is erroring out:

```
/usr/bin/ld: resumedatasavingmanager.o: in function `int libtorrent::detail::bencode_recursive<Utils::IO::FileDeviceOutputIterator>(Utils::IO::FileDeviceOutputIterator&, libtorrent::entry const&)':
resumedatasavingmanager.cpp:(.text._ZN10libtorrent6detail17bencode_recursiveIN5Utils2IO24FileDeviceOutputIteratorEEEiRT_RKNS_5entryE[_ZN10libtorrent6detail17bencode_recursiveIN5Utils2IO24FileDeviceOutputIteratorEEEiRT_RKNS_5entryE]+0x407): undefined reference to `libtorrent::entry::dict[abi:cxx11]() const'
/usr/bin/ld: tracker.o: in function `BitTorrent::Tracker::prepareAnnounceResponse(BitTorrent::Tracker::TrackerAnnounceRequest const&)':
tracker.cpp:(.text+0x1c0d): undefined reference to `libtorrent::entry::entry(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, libtorrent::entry, libtorrent::aux::strview_less, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, libtorrent::entry> > >)'
/usr/bin/ld: tracker.cpp:(.text+0x2288): undefined reference to `libtorrent::entry::entry(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, libtorrent::entry, libtorrent::aux::strview_less, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, libtorrent::entry> > >)'
/usr/bin/ld: tracker.o: in function `BitTorrent::Tracker::processRequest(Http::Request const&, Http::Environment const&) [clone .cold]':
tracker.cpp:(.text.unlikely+0xb1a): undefined reference to `libtorrent::entry::entry(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, libtorrent::entry, libtorrent::aux::strview_less, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, libtorrent::entry> > >)'
/usr/bin/ld: tracker.o: in function `int libtorrent::detail::bencode_recursive<std::back_insert_iterator<QByteArray> >(std::back_insert_iterator<QByteArray>&, libtorrent::entry const&)':
tracker.cpp:(.text._ZN10libtorrent6detail17bencode_recursiveISt20back_insert_iteratorI10QByteArrayEEEiRT_RKNS_5entryE[_ZN10libtorrent6detail17bencode_recursiveISt20back_insert_iteratorI10QByteArrayEEEiRT_RKNS_5entryE]+0x230): undefined reference to `libtorrent::entry::dict[abi:cxx11]() const'
/usr/bin/ld: tracker.o: in function `void std::vector<libtorrent::entry, std::allocator<libtorrent::entry> >::_M_realloc_insert<std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, libtorrent::entry, libtorrent::aux::strview_less, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, libtorrent::entry> > >&>(__gnu_cxx::__normal_iterator<libtorrent::entry*, std::vector<libtorrent::entry, std::allocator<libtorrent::entry> > >, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, libtorrent::entry, libtorrent::aux::strview_less, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, libtorrent::entry> > >&)':
tracker.cpp:(.text._ZNSt6vectorIN10libtorrent5entryESaIS1_EE17_M_realloc_insertIJRSt3mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES1_NS0_3aux12strview_lessESaISt4pairIKSB_S1_EEEEEEvN9__gnu_cxx17__normal_iteratorIPS1_S3_EEDpOT_[_ZNSt6vectorIN10libtorrent5entryESaIS1_EE17_M_realloc_insertIJRSt3mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES1_NS0_3aux12strview_lessESaISt4pairIKSB_S1_EEEEEEvN9__gnu_cxx17__normal_iteratorIPS1_S3_EEDpOT_]+0x130): undefined reference to `libtorrent::entry::entry(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, libtorrent::entry, libtorrent::aux::strview_less, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, libtorrent::entry> > >)'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:1039: qbittorrent] Error 1
make[1]: Leaving directory '/builddir/qbittorrent-4.2.5/build-x11/src'
make: *** [Makefile:47: sub-src-make_first] Error 2
```

**Comment by ericonr** [comment]
Needs a confirmation if `deluge` requires a rebuild or not.

**Comment by pullmoll** [comment]
I'd say we revbump pkgs depending on libtorrent-rasterbar-devel in any case.

**Comment by ericonr** [comment]
Will do.

**Comment by ericonr** [comment]
@pullmoll done!

File: srcpkgs/qbittorrent/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'qbittorrent'
 pkgname=qbittorrent
 version=4.2.5
-revision=2
+revision=4
```
Comment (pullmoll) [inline srcpkgs/qbittorrent/template:4]: Did we already have revision=3?

Comment (ericonr) [reply srcpkgs/qbittorrent/template]: No, sorry. Fixed.

**Comment by daniel-eys** [comment]
Looks like upstream libtorrent-rasterbar updated their m4 macros during 1.2.8 -> 1.2.9 which resulted in a makefile with `-std=c++11` and a silent abi breakage with C++14 packages like qbittorrent.

Unfortunately, this slipped through my tests when I was doing the recent libtorrent-rasterbar pr as I was only testing with libtorrent-rasterbar-examples and btfs (C++11).

Rebuilding current libtorrent-rasterbar with C++14 should potentially fix this.
Anyway, let's revert and wait for upstream.

**Pull request merged**